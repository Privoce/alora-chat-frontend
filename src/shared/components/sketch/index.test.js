import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	SketchComponent
} from 'shared/components';

describe('<SketchComponent />', () => {
	const modelI = {
		circle: true,
		width: 40,
		height: 40
	};

	const getWrapper = (model) => {
		return shallow((
			<SketchComponent {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
