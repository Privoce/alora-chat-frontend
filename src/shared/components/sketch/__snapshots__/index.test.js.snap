// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SketchComponent /> renders correctly 1`] = `
<div
  className="sketch-wrapper"
>
  <div
    className="sketch circle"
    style={
      Object {
        "height": 40,
        "margin": undefined,
        "width": 40,
      }
    }
  />
</div>
`;
